src/main/resources/templates/team_page_join_team.html

```diff
@@ -15,8 +15,8 @@ <h2 class="team-heading team-heading-current-team" th:text="#{joinTeam.header}">
                 <form  method="post" action="#" th:action="@{/teams/join_team}" th:object="${teamPageJoinTeamForm}" role="form">
                     <fieldset>
                         <div class="form-group" th:classappend="${#fields.hasErrors('teamName')}? 'has-error'">
-                            <input type="text" class="form-control" id="inputTeam" placeholder="Team Name to Join" name="" th:field="*{teamName}"/> 
-                            <span class="help-block" th:if="${#fields.hasErrors('teamName')}" th:errors="*{teamName}">Team name cannot be empty</span>
+                            <input type="text" class="form-control" id="inputTeam" placeholder="Team Name to Join" name="" th:field="*{teamName}"/>
+                            <span class="account-change-fail" th:if="${#fields.hasErrors('teamName')}"><i class="fa fa-exclamation-circle"></i> <span th:errors="*{teamName}"></span></span>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)